src/index.html

```diff
@@ -4,8 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Gallery</title>
+    <script src="//cdn.auth0.com/js/lock/10.1.0/lock.min.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```

Comment:
there's already a `<meta>` tag on line 6. figure out which one is the correct one to use.